From 49e984d364f8ed4bbdf1a9a3dacee08e7b25453e Mon Sep 17 00:00:00 2001
From: Andy Whitcroft <apw@canonical.com>
Date: Fri, 2 Jan 2015 15:41:12 +0000
Subject: [PATCH 09/33] UBUNTU: apparmor -- follow change to this_cpu_ptr

Signed-off-by: Andy Whitcroft <apw@canonical.com>
Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
---
 security/apparmor/include/path.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/apparmor/include/path.h b/security/apparmor/include/path.h
index 09b47b8..79c48d4 100644
--- a/security/apparmor/include/path.h
+++ b/security/apparmor/include/path.h
@@ -64,7 +64,7 @@ DECLARE_PER_CPU(struct aa_buffers, aa_buffers);
 #define __get_buffer(N) ({					\
 	struct aa_buffers *__cpu_var; \
 	AA_BUG_PREEMPT_ENABLED("__get_buffer without preempt disabled");  \
-	__cpu_var = &__get_cpu_var(aa_buffers);			\
+	__cpu_var = this_cpu_ptr(&aa_buffers);			\
         __cpu_var->buf[(N)]; })
 
 #define __get_buffers(X...)		\
-- 
2.7.4

