From 474abfb5460dcd4af5e651d12631b87d36c5fc66 Mon Sep 17 00:00:00 2001
From: Simon Fels <simon.fels@canonical.com>
Date: Mon, 27 Mar 2017 14:18:41 +0200
Subject: [PATCH 3/4] cmd: disable check for xfs/xqm.h

---
 cmd/configure.ac | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/cmd/configure.ac b/cmd/configure.ac
index 3c921ac..a6b62b3 100644
--- a/cmd/configure.ac
+++ b/cmd/configure.ac
@@ -19,7 +19,8 @@ AC_LANG([C])
 # Checks for header files.
 AC_CHECK_HEADERS([fcntl.h limits.h stdlib.h string.h sys/mount.h unistd.h])
 AC_CHECK_HEADERS([sys/quota.h], [], [AC_MSG_ERROR(sys/quota.h unavailable)])
-AC_CHECK_HEADERS([xfs/xqm.h],   [], [AC_MSG_ERROR(xfs/xqm.h unavailable)])
+# YOCTO: Disabled until we have a better way to do this check.
+# AC_CHECK_HEADERS([xfs/xqm.h],   [], [AC_MSG_ERROR(xfs/xqm.h unavailable)])
 
 # Checks for typedefs, structures, and compiler characteristics.
 AC_CHECK_HEADER_STDBOOL
-- 
2.7.4

