From 6951a437b889ed4ca1d8d8c0235205ce31f19c9c Mon Sep 17 00:00:00 2001
From: Simon Fels <simon.fels@canonical.com>
Date: Fri, 22 Jul 2016 10:22:22 +0200
Subject: [PATCH] Disable doc generation

---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 0b7df92..51be5e4 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,2 +1,2 @@
-SUBDIRS = src tests docs
+SUBDIRS = src tests
 EXTRA_DIST = PORTING README.md
-- 
2.7.4

